src/cmr/sizing/formats.clj

```diff
@@ -14,10 +14,16 @@
     (read-string value)
     value))
 
+(defn- get-dimensionality
+  "Get the dimensionality, which is the multiplication of all the sizes
+  in all the Dimensions."
+  [variable]
+  (let [dims (get-in variable [:umm :Dimensions])]
+    (reduce * (map :Size dims))))
```

Comment:
I can't find any validation on Size for umm var dimensions that makes sure its greater than zero, or not zero.  It probably wouldn't affect the total estimate too much but it might be something we want to protect against.